db/dump.py

```diff
@@ -273,6 +327,11 @@ def import_db_dump(archive_path):
         connection.close()
 
     pxz.stdout.close()
+    pxz.wait()
+
+    logging.info('Updating sequences...')
```

Comment:
leaving this rn for consistency, I'll open a seperate PR for converting all these logging calls to use the flask logger.

Reply:
[AB-366](https://tickets.metabrainz.org/browse/AB-366)